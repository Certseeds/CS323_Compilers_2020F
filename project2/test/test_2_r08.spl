int compare1(int x, int y)
{
  if (x > y) {return 1;}
  else if (y > x) {return -1;}
  else {return 0;}
}
float compare2(int a, int b)
{
  if (b > a) {return -1.0;}
  if (b == a) {return 0.0;}
  return 1.0;
}
int test_2_r08(int i, int m, int n)
{
  if (i == 0)
  {
    return compare1(m, n);
  }
  else
  {
    return compare2(m, n);
  }
}
