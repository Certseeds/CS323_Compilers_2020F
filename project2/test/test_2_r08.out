Error type 8 at Line 21: incompatiable return type
